_posts/2020-07-14-what-is-ricing.md

```diff
@@ -0,0 +1,56 @@
+	Ricing follows the same principle, make your system do and look like however you want. Don't like how it looks? Redo it or maybe change a few aspects! You have so many choices that many a days go by when I start ricing. It could be the simplest or the most complex, but it all depends on you. It is what you want to do. 
+	The philosophy of ricing , at least this is what I believe, is that you have all the freedom to make your system the way you want. People who are indifferent to it, at least give it a try. I promise you won't regret it. 
+
+How do I rice?
+--------------
+To answer this, I'll give a rough overview. Albeit to common suggestion, you don't have to necessarily use Arch Linux, Void Linux, Nix OS, Gentoo, or BSDs to rice your systems. These distros have amazing user contributed packages and/or are not traditionally installed. Most of these are customized from the very beginning, you get to choose what you put in your system and you aren't given anything but a plaform to start building. In this case, the distros are the platforms and the packages are a mall of all lego blocks and you get to build whatever you want with it. 
+	I wouldn't recommend getting started with these if you really just want to try out ricing and not leave the comfort of your distro. Just stick with what you like. If you don't have a user built package, you'd 100% have github repo that is made from source.
+			But if you are still searching for a perfect distro I'd recommend trying out one of these. Preferably Arch Linux (BTW I use arch).
+
+After you have a distro or decide to stick with your own, you can customize anything you want. I'd recommend to start with the following first:
+- Shell: Most people start with bash. I'd suggest zsh and oh-my-zsh themes.
+- Terminal Emulator: There are many terminal emulators. Just pick one from the pletora of existing ones. I'd recommend starting with a drop-down terminal, that drops from the top of your screen and is binded to a key for ease of access. KDE has Yakuake, GNOME users install guake and XFCE users just keybind `xfce-terminal --drop-down` to any key you want.
+- WM: Window Managers if you want more fancy shortcuts. 
+- Themes: just change the color scheme and wallpaper
+
+You have a pletora of options. I'd suggest exploring [r/unixporn](www.reddit.com/r/unixporn) for inspirations and tutorials or their rice configurations files. These files are the best way to start off as you'd have a base to start with instead of doing everything yourself.
```

Comment:
Few grammatical  errors caught here:
- a small typo here: it's "plethora"
- for *inspiration* (as a singular)
- *configuration* (again as a singular)

Reply:
I'm so sorry for me terrible grammar. Thanks for pointing out.